tools/__tasks__/compile/conf/clean.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const rimraf = require('rimraf');
+
+const {root} = require('../../config').paths;
+
+module.exports = {
+    description: 'Clear template rendering artefacts',
+    task: () => rimraf.sync(path.resolve(root, 'common', 'conf', 'assets'))
```

Comment:
I wish it was really obvious that this dir was the output of a copy task, an svg build task and one-half of an uglify task. The folder org in `__tasks__` gives you a much better chance of seeing the dependencies, so excellent work!

My thought just now was that every task should have a clean and a build, ideally. For example, every `index.js` has both a `task` and a `clean` key. Such ideals probably won't survive in our codebase though.